default.nix

```diff
@@ -0,0 +1,204 @@
+      buildInputs = [ lerfanva xclip ];
+    
+      installPhase = ''
+        mkdir -p $out/bin
+
+        ln -s ${lerfanva}/bin/lerfanva $out/bin/lerfanva
+
+        cat << EOF > $out/bin/lerfanva-clip
+        #!/usr/bin/env bash
+        SEL=0
+        case \$1 in
+          1) SEL=primary;;
+          2) SEL=secondary;;
+          *) SEL=clipboard;;
+        esac
+
+        OUT=\$(${lerfanva}/bin/lerfanva <(echo \$(xclip -o -selection \$SEL)))
+        echo -n \$OUT | ${xclip}/bin/xclip -selection \$SEL
+        echo \$OUT
+        EOF
+
+        chmod +x $out/bin/lerfanva-clip
+      '';
+
+      meta = with stdenv.lib; {
+        version = "0.1.0";
+        description = "Replace clipboard or selection content with its transliteration into Zbalermorna";
+        homepage = https://github.com/jackhumbert/zbalermorna;
+        platforms = platforms.linux;
+        license = licenses.agpl3;
+      };
+    };
+
+    mkFontDrv = name: attrs: stdenv.mkDerivation {
+      inherit name;
+      inherit (attrs) version;
+      src = ./.;
+      buildInputs = [ fontforge ];
+      installPhase = ''
+        mkdir -p $out/share/fonts/zbalermorna
+        ./build.sh ${name}
+        cp fonts/${name}.otf $out/share/fonts/zbalermorna
+      '';
+
+      meta = {
+        inherit (attrs)
+          version
+          license;
+        description = "A font for the Zbalermorna writing system of the language Lojban";
+        homepage = https://github.com/jackhumbert/zbalermorna;
+        platforms = stdenv.lib.platforms.linux;
+      };
+    };
+
+    table-zbalermorna = let version = "0.1.0"; in with ibus-engines; stdenv.mkDerivation {
+      name = "table-zbalermorna";
+      inherit version;
+      src = ./.;
+      buildInputs = [ table ];
+      buildPhase = ''
+        export HOME=$TMP
+        ${table}/bin/ibus-table-createdb -n zlm.db -s ime/zlm.ibus
+      '';
+      installPhase = ''
+        TABLE_LOCATION=$out/share/ibus-table/tables
+        mkdir -p $TABLE_LOCATION
+        cp zlm.db $TABLE_LOCATION/zlm.db
+        # TODO: Add icon? What icon to use? Where to get it from? Where do you put it?
+        # cp assets/img/favicon.png $out/share/ibus-table/???
+      '';
+
+      meta = with stdenv.lib; {
+        isIbusEngine = true;
+        inherit version;
+        description = "Basic input method for Zbalermorna text";
+        homepage = https://github.com/jackhumbert/zbalermorna;
+        platforms = platforms.linux;
+        license = licenses.lgpl3;
+      };
+    };
+
+    fonts = builtins.mapAttrs mkFontDrv (with stdenv.lib.licenses; {
+      # I extracted versions with: grep -oE '^Version: .*$' src/*.sfd
+      balvi-regular = {
+        version = "1.001";
+        license = ofl;
+      };
+      crisa-light = {
+        version = "2.015";
+        license = ofl;
+      };
+      crisa-regular = {
+        version = "0.1";
+        license = ofl;
+      };
+      dunda-regular = {
+        version = "1.10";
+        license = unfree;
```

Comment:
Are you sure? The font description (at least as stated in font manager when imported) says it's modified from Lato family, which is SIL OFL, and the license is copyleft

Reply:
If so then I'll happily change it to OFL! I just couldn't find any explicitly stated license for it at the time.

Reply:
I mean I hope you're right (too lazy to verify myself) but what makes me hesitate is just that it's supposedly[^1] based on [Tengwar Annatar](https://www.dafont.com/tengwar-annatar.font) which I can't find a license for. Is that one also modified Lato?

I really don't mind eagerly enforcing copyleft, so I'll take your suggestion for now.

[^1]: stated here: https://jackhumbert.github.io/zbalermorna/fonts

Reply:
We can kindly ask @Alatius what is the current license of the Tengwar Annatar and if it is not SIL OFL, if it it is possible to license it as such

Reply:
Reading the pdf included in the files for Tengwar Annatar it explicitly says freeware, copyright belonging to @Alatius. If it can't be open sourced, then the only way Dunda may be considered OFL is if was merely *inspired* by it but actually a modification of Lato. I don't think "inspired by" is grounds for copyright infringement.